schemas/asn1-lib-module-schema.json

```diff
@@ -24,7 +24,7 @@
       "properties": {
         "from": { "$ref": "#/definitions/asn1TypeName" },
         "types": {
-          "types": "array",
+          "type": "array",
```

Comment:
Shouldn't the change be performed in line 59?

Reply:
ok, I see the issue now :)

Reply:
yep, one has to get use to reading json schema ;)